teuthology/kill.py

```diff
@@ -239,7 +239,7 @@ def nuke_targets(targets_dict, owner):
         stdout=subprocess.PIPE,
         stderr=subprocess.STDOUT)
     for line in iter(proc.stdout.readline, ''):
-        line = line.replace('\r', '').replace('\n', '')
+        line = line.encode().replace('\r', '').replace('\n', '')
```

Comment:
might want to put
```py
line = line.encode().replace(b'\r', b'').replace(b'\n', b'')
```

Reply:
nice catch... need to double check

Reply:
I've dropped this commit, and give it a try in another PR.